plugins/infinispan70/src/main/java/org/radargun/service/InfinispanClientListeners.java

```diff
@@ -92,8 +92,8 @@ public void removeExpiredListener(String cacheName, ExpiredListener listener, bo
    public void removeListener(String cacheName, boolean sync) {
       GenericClientListener listener = getListenerOrThrow(cacheName, sync);
       if (listener.isEmpty()) {
-         service.getRemoteManager(false).getCache(cacheName).removeClientListener(listener);
-         service.getRemoteManager(true).getCache(cacheName).removeClientListener(listener);
+         service.getRemoteManager(false).getCache(cacheName == null ? "" : cacheName).removeClientListener(listener);
```

Comment:
getCache() has the same meaning as get.Cache(""). You can find it in RCM source code. I can change it but that's just a cosmetic change with twice as many lines of code.